### ISSUE TITLE
Add power status display for telescope

### ISSUE THREAD

**Issue by DavidLP** [SiLab-Bonn/silab_online_monitor#5]
It would be beneficial to have an overview of all voltages and currents of the telescope.
This requires the following:

1. Measuring and sending the info as a scan parameter in the pyBAR scan
2. Adding a telescope status converter in silab_online_monitor to filter this info from the data stream
3. Adding a receiver for the data stream with nice plots over time, maybe similar to the [Beast TDC display](https://github.com/SiLab-Bonn/silab_online_monitor/blob/development/silab_online_monitor/receiver/beast_tdc_receiver.py)

**Comment by YannickDieter** [comment]
So the voltages/currents should then stored in the scan.h5 file or in a seperated one?
Pascal added already a status dock, so I can use this 👍

**Comment by DavidLP** [comment]
Yes, if it is a scan parameter the values are automatically stored in the scan.h5 file, thus also automatically available in the data stream for the online monitor

**Comment by leloup314** [comment]
Is it possible to obtain power consumption details (eg. voltage, current) for every single plane of the telescope (m26 and fei4) ?

**Comment by themperek** [comment]
> Is it possible to obtain power consumption details (eg. voltage, current) for every single plane of the telescope (m26 and fei4) ?

**Comment by themperek** [comment]
>Is it possible to obtain power consumption details (eg. voltage, current) for every single plane of the telescope (m26 and fei4) ?

NO is easy answer. You can monitor power supplies. One could also use some special circuit on MMC3 board but...

**Comment by leloup314** [comment]
So what we can monitor is input voltage/current of Mimosas and VDDD and VDDA and corresponding currents for FE-I4, correct?

**Comment by DavidLP** [comment]
Yes, one power supply for Mimosa planes and one for the FE-I4 we can monitor. And that is to my mind sufficient.

**Comment by DavidLP** [comment]
This is implemented and working, no?

**Comment by leloup314** [comment]
Yes, in principal this is working. The `development` version of the `telescope_status` converter tough, replays simulated data right now, which is very confusing during a test beam. @YannickDieter can you maybe commit and push your version of the `telescope_status` converter? This should be correct since you used this feature in a test beam before, correct?

**Comment by YannickDieter** [comment]
This has been changed a lot. Due to the new readout software for M26 there are no scan parameters anymore and we separated FE-I4 and M26. I made a general status monitor (+ producer and converter) were one can define any remotely accessible device (power supply, source meter, ...) for monitoring its data. But I have to work on this a bit. I will this push it if its done ;)
